Algorithms/Sorting/Java/BucketSort.java

```diff
@@ -0,0 +1,62 @@
+//Bucket Sort in java
+
+import java.util.*;
+
+
+public class BucketSort {
+
+
+     //Driver method
+	public static void main(String[] args) {
+		
+
+        double a[] = {0.33,0.756,0.146,0.94,0.32,0.545};
+		double res[] = new double[a.length];
+		
+		//Calling a function named buket.
+		 res =	bucket(a,a.length);
+
+		//Printing the sorted array. 
+       for(int i=0;i<res.length;i++)
```

Comment:
This line is not indented properly. Make sure the indentation in the entire code is maintained properly.